src/autocomplete/AutocompleteViewWrapper.js

```diff
@@ -39,8 +40,10 @@ export default class AutocompleteViewWrapper extends PureComponent<Props> {
       onMessageAutocomplete,
       onTopicAutocomplete,
     } = this.props;
+    const { height } = Dimensions.get('window');
+
     return (
-      <View style={[styles.wrapper, { marginBottom }]}>
+      <View style={[styles.wrapper, { height: height / 2 - marginBottom, marginBottom }]}>
```

Comment:
(Before we had it as 1/4 of height, to tackle landscape scenario too b4cd7c829a7d17888902f5468d4ea0a0f1084c42)